app/.env.example

```diff
@@ -39,7 +39,7 @@ TF_GH_REPO=
 TF_MODULE_GH_REF=
 GH_ACCESS_TOKEN=
 GH_API_TOKEN=
-DATABASE_URL=
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/realm_registry
```

Comment:
I would change this to:
`postgresql://<DBUSER>:<DBPWD>@localhost:5432/realm_registry`